### PR TITLE
Update 5.10.0.md

### PR DESCRIPTION
**Pull request by luhenry**

### CODE REVIEW

File: docs/about-mono/releases/5.10.0.md [hunk 1]
```diff
@@ -211,7 +214,17 @@ Bug Fixes
 * GitHub [#6777](https://github.com/mono/mono/issues/6777) - Memory corruption in Mono 5.8.0.108.
 * GitHub [#6848](https://github.com/mono/mono/issues/6848) - netstandard facade dlls have different public key tokens
 * GitHub [#6848](https://github.com/mono/mono/issues/6948) - [XI]WatchOS apps fail to build with error "Undefined symbols for architecture armv7k" using XI with mono-2017-12 support
+* GitHub [#6940](https://github.com/mono/mono/issues/6940) - DNS lookup of IPv6-only hosts fails
 * GitHub [#7086](https://github.com/mono/mono/issues/7086) - Hybrid AOT throws invalid cast exception on dictionaries indexed with enum
+* GitHub [#7093](https://github.com/mono/mono/issues/7093) - Regression in Array interface conversions
+* GitHub [#7095](https://github.com/mono/mono/issues/7095) - Assertion at class-accessors.c:31, condition `mono_class_is_ginst (klass)' not met
```
Comment (akoeplinger) [inline docs/about-mono/releases/5.10.0.md]: please escape the backtick ` here

File: docs/about-mono/releases/5.10.0.md [hunk 2]
```diff
@@ -166,6 +168,7 @@ Bug Fixes
 * Bugzilla [#57938](https://bugzilla.xamarin.com/show_bug.cgi?id=57938) - Differences in behavior Type.Fullname in Mono and .NET Framework when Type is a generic type that is not a generic type definition but contains unresolved type parameters
 * Bugzilla [#58400](https://bugzilla.xamarin.com/show_bug.cgi?id=58400) - System.Uri broken with files
 * Bugzilla [#58411](https://bugzilla.xamarin.com/show_bug.cgi?id=58411) - System.Security.Cryptography.CryptographicException: Store root doesn't exist
+* Bugzilla [#58413](https://bugzilla.xamarin.com/show_bug.cgi?id=58413) - Environment.Tickcount is changed when the date is changed in android
```
Comment (akoeplinger) [inline docs/about-mono/releases/5.10.0.md]: The fix for this was reverted from 2017-12 so we should remove this.

**Review by akoeplinger — DISMISSED** [review]

**Review by akoeplinger — APPROVED** [review]

**Comment by marek-safar** [comment]
@luhenry I think we should report the new dot version fixes more visibly maybe even with line separator not just merge it with the existing list

**Comment by akoeplinger** [comment]
I'd prefer even adding separate release notes for .PATCH releases especially since we agreed that we'll bump .PATCH once a release hits the stable channel.

**Pull request merged**